runtime/verbose/errormessagehelper.c

```diff
@@ -857,8 +857,12 @@ mapDataTypeToUTF8String(J9UTF8Ref* dataType, StackMapFrame* stackMapFrame, Metho
 
 			dataType->bytes = methodInfo->signature.bytes + typeValue;
 			dataType->length = typeLength;
-			/* Ignore 'L' and ';' to get the full string of argument in signature */
-			if ('L' == *dataType->bytes) {
+			/* Ignore ('L' or 'Q') and ';' to get the full string of argument in signature */
+			if (('L' == *dataType->bytes)
+#if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)
+				|| ('Q' == *dataType->bytes)
+#endif /* J9VM_OPT_VALHALLA_VALUE_TYPES */
+			) {
```

Comment:
@tajila you mentioned a different part of `errormessagehelper.c` should not be changed. Should this be reverted as well?

Reply:
yes please remove this change